theme-utils.mjs

```diff
@@ -95,69 +99,75 @@ async function pushButtonDeploy(repoType) {
 	}
 
 	try {
-		if (repoType === 'git' ) {
-			await cleanSandboxGit();
-		}
-		else {
-			await cleanSandboxSvn();
-		}
+		await deployThemesByRepo( repoType, 'pub' );
+		await deployThemesByRepo( repoType, 'premium' );
+	}
+	catch (err) {
+		console.log("ERROR with deply script: ", err);
+	}
+}
 
-		let hash = await getLastDeployedHash();
-		let diffUrl;
+async function deployThemesByRepo( repoType, repo ) {
+	if (repoType === 'git' ) {
+		await cleanSandboxGit( repo );
+	}
+	else {
+		await cleanSandboxSvn( repo );
+	}
 
-		await versionBumpThemes();
+	let hash = await getLastDeployedHash( repo );
+	let diffUrl;
 
-		let changedThemes = await getChangedThemes(hash);
+	await versionBumpThemes( repo );
 
-		await pushChangesToSandbox();
+	let changedThemes = await getChangedThemes(hash);
 
-		await updateLastDeployedHash();
+	await pushChangesToSandbox( repo );
```

Comment:
As noted elsewhere this will still push ALL the changes in the changeset between HASH...HEAD to the given repository.  Copying and deleting everything from both repositories.  Which isn't very helpful.